app/policies/article_policy.rb

```diff
@@ -19,4 +19,8 @@ def show?
     true
   end
 
+  def like?
+    create?
```

Comment:
validate here based on `show?`. logic is that if they can see the article, they should also be able to like it.